test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -694,7 +696,8 @@ func BenchmarkPerfScheduling(b *testing.B) {
 					for feature, flag := range tc.FeatureGates {
 						defer featuregatetesting.SetFeatureGateDuringTest(b, utilfeature.DefaultFeatureGate, feature, flag)()
 					}
-					results := runWorkload(ctx, b, tc, w, false)
+					informerFactory, client, dyncClient := setupSchedulerForWorkload(ctx, b, tc.SchedulerConfigPath, tc.FeatureGates)
```

Comment:
Maybe next is we can setup the api-server first then only recreate the scheduler with each test. This differs from integration tests with benchmark test.

Reply:
I'm not sure what you are suggesting. Can you rephrase and explain what benefit it has?

Reply:
I'm still confused...

`TestScheduling` already reuses the same apiserver and scheduler as much as possible. `BenchmarkPerfScheduling` cannot reuse them because etcd gets wiped by restarting it.

Reply:
We'll create the api-server inside the `mustSetupScheduler`, but actually we can preserve the api-server and re-create the scheduler for each case, the same as we did in https://github.com/kubernetes/kubernetes/pull/116634

Reply:
@kerthcet: are you talking about the integration testing?

Remember that `mustSetupScheduler` only gets called once per unique configuration. Currently there are two of those, with and without the DRA feature gate. We cannot preserve the api-server when switching from one to the other, because the feature gate also affects it.